src/compiler/checker.ts

```diff
@@ -36458,10 +36485,31 @@ namespace ts {
             return literalTypeToNode(<FreshableType>type, node, tracker);
         }
 
-        function getJsxFactoryEntity(location: Node) {
+        function getJsxFactoryEntity(location: Node): EntityName | undefined {
             return location ? (getJsxNamespace(location), (getSourceFileOfNode(location).localJsxFactory || _jsxFactoryEntity)) : _jsxFactoryEntity;
         }
 
+        function getJsxFragmentFactoryEntity(location: Node): EntityName | undefined {
+            if (location) {
+                const file = getSourceFileOfNode(location);
+                if (file) {
+                    if (file.localJsxFragmentFactory) {
+                        return file.localJsxFragmentFactory;
+                    }
+                    const jsxFragPragmas = file.pragmas.get("jsxfrag");
+                    const jsxFragPragma = Array.isArray(jsxFragPragmas) ? jsxFragPragmas[0] : jsxFragPragmas;

```

Comment:
Hmm, for consistency, just use the internal `isArray` helper here.

Reply:
done